NAMESPACE_old

```diff
@@ -0,0 +1,17 @@
+export(NCT)
```

Comment:
this file can be deleted :slightly_smiling_face:

Reply:
file deleted